docs/ja/api-reference/ngsiv2.md

```
@@ -250,11 +257,14 @@ DELETE /v2/entities/{entityId}
 
 | パラメータ | 型 | 説明 |
 |-----------|------|-------------|
-| `type` | string | エンティティタイプ |
+| `type` | string | エンティティ型 |
+
+**レスポンス**: `204 No Content`
```

markdownlint-cli2 (0.22.1)

[warning] Heading style
Expected: atx; Actual: setext

(MD003, heading-style)